xcodeproj.gemspec

```diff
@@ -23,6 +23,7 @@ Gem::Specification.new do |s|
 
   s.add_runtime_dependency 'activesupport', '>= 3'
   s.add_runtime_dependency 'colored',       '~> 1.2'
+  s.add_runtime_dependency 'claide',        '~> 0.8.2'
```

Comment:
Should be 0.9.0 I guess since it will need https://github.com/CocoaPods/CLAide/pull/49. Let's make sure this is correct once CLAide is released.